src/pybind/mgr/influx/module.py

```diff
@@ -45,6 +45,8 @@ class Module(MgrModule):
         'username': None,
         'password': None,
         'interval': 5
+        'ssl': 'false',
+        'verify_ssl': 'false',
```

Comment:
Shouldn't they both be a boolean instead of a string?

Reply:
The `get_config` command returns strings and the influxdb-client expects bool.
In every case I need to cast to bool.
If I would set the defaults to `True` I would need one more cast, because:
```python
True == 'true'
False
```